README.md

```diff
@@ -111,4 +117,4 @@ If clients hang in an "updating" state, be sure to check the following:
 ---
 
 ##### [Official MTProxy GitHub Repository](https://github.com/TelegramMessenger/MTProxy)
-
+##### [Forked project from MTProxy with patches](https://github.com/GetPageSpeed/MTProxy)
```

Comment:
```suggestion
add as alternatives
```

Reply:
Done, I documented a bit more the environments with this second repository as an usage example.